docs/transport/transport.md

```diff
@@ -0,0 +1,60 @@
+# gittuf Remote Transfer Protocol
+
+Last Modified: July 2, 2024
+
+## Introduction
+
+Alongside the `gittuf` binary, gittuf ships with a custom remote transfer
+protocol binary, implementing Git's [remote-helper
+interface](https://git-scm.com/docs/gitremote-helpers). We call this the
+**transport** binary, named `git-remote-gittuf`.
+
+It's an easy way to get started with using gittuf on your repository, as it
+takes care of the following common operations for you:
+
+- Creating RSL entries upon pushing your changes
+- Fetching gittuf metadata when pulling changes
+
+> [!NOTE] The transport does not perform the steps needed to *initialize* a
+> gittuf repository (i.e. setting up root of trust, policy, etc.). These steps
+> must be done manually for new repositories (see the [getting started
+> guide](/docs/get-started.md)).
+
+The gittuf transport supports both HTTPS and SSH remotes.
+
+## How to Obtain
+
+Currently, the transport must be built from source. Running `go install` will
+compile the transport and place it in your `GOBIN`, which is the same location
+that the `gittuf` binary is installed into.
+
+## How to Use
+
+Once it's installed, using the custom transport is simple, and depends on the
```

Comment:
Same thing here, I'd make this more generic and say in either case, we want the prefix prepended to the URL. Then the two subsecs.